modules/unsupported/mbstyle/src/main/java/org/geotools/mbstyle/layer/MBLayer.java

```diff
@@ -301,6 +325,47 @@ public JSONObject paint(){
     public JSONObject getPaintProperties(){
         return new JSONObject();
     }
+
+    /**
+     * Transforms a given {@link MBLayer} to a GeoTools {@link FeatureTypeStyle}.
+     *
+     * @param layer The MBLayer to transform.
+     * @param minScaleDenominator Used to determine zoom level restructions for generated rules
+     * @param maxScaleDenominator Used to determine zoom level restructions for generated rules
+     * @return A feature type style from the provided layer.
+     */
+    public FeatureTypeStyle transform(MBStyle styleContext, Double minScaleDenominator, Double maxScaleDenominator) {
```

Comment:
we are mixing abstractions in these parameters, MBStyle is map box style, min/max scale denominator is SLD. Consider changing parameters to zoomLevel for consistency.